ArticleTemplates/assets/js/boot.js

```diff
@@ -158,6 +158,37 @@ const init = opts => {
                 common.init();
             });
         }
+
+        const listenForEmailEmbedIFrameResize = () => {
+            const allIframes = [].slice.call(
+                document.querySelectorAll('.email-sub__iframe')
+            );
+            window.addEventListener('message', (event) => {
+                const iframes = allIframes.filter((i) => {
+                    try {
+                        return i.contentWindow === event.source;
+                    } catch (e) {
+                        return false;
+                    }
+                });
+                if (iframes.length !== 0) {
+                    try {
+                        const message = JSON.parse(event.data);
```

Comment:
Would it be a good idea to verify the structure of this message? E.g. making sure that the `value` is definitely a simple number?

https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage#security_concerns

Reply:
I have a feeling that some messages come as a string, e.g `'500px'`. Although this works in the current implementation, it's not very well documented whether the `height` attr will always accept a string like that.

I will validate the possible values of `value`. If they are all simple numbers, then it seems reasonable to add the check here too